packages/nimble-components/src/toggle-button/styles.ts

```diff
@@ -10,23 +15,64 @@ export const styles = css`
     .control[aria-pressed='true'] {
         background-color: ${fillSelectedColor};
         border-color: ${fillSelectedColor};
+        position: relative;
     }
 
     .control[aria-pressed='true']:hover {
```

Comment:
When I hover over a toggle button in the pressed state, the background is changing to appear unpressed. It looks like this is a regression, likely caused by removing background-color here.

Same issue with focusVisible below.


https://github.com/ni/nimble/assets/10500124/9ef8df2e-b682-4beb-9402-01e5ae071472

Reply:
I looked into it, it's because control::before sets the z-index to -1, so the background that should show up on :active::before is behind the background of the entire matrix story. I'm looking into ways that I can fix this without having to set the z-index to 0, since doing so affects the text color.